modules/web/js/ballerina/ast/connector-action.js

```diff
@@ -441,5 +441,18 @@ define(['lodash', './node', 'log', '../utils/common-utils'], function(_, ASTNode
         return !_.isNil(connectorReference) ? connectorReference : this.getParent(). getConnectorByName(connectorName);
     };
 
+    /**
+     * Get all the connector references in the immediate scope
```

Comment:
when we have a connector action, say in a resource, and the connector action arrow should give the priority to the resource level connector. If there is no resource level connector it should connect to the service level connector